_includes/overview/160.html

```diff
@@ -0,0 +1,6 @@
+<div class="container">
+<p class="lead font-weight-normal">
+<br/>
+{{ locale.leap_16_coming_soon }}
```

Comment:
https://jekyllrb.com/docs/liquid/filters/#markdownify, so that you don't need to use html in the locale file. You can also write this entire file as a markdown document instead, by renaming it to 160.md